examples/docs_snippets/docs_snippets/concepts/assets/asset_group_module.py

```diff
@@ -0,0 +1,19 @@
+# isort: skip_file
```

Comment:
This is *definitely* a quick and dirty example. If ya have something better, please feel free to add it in.

Reply:
this works great, although I'll flag that it's probably best to show the import code as well so people know what type of thing `cereal` is.

this would be:

```
import my_package.cereal as cereal

cereal_assets = load_assets_from_package_module(
    cereal, group_name="cereal_assets"
)
```

not a huge deal, just thought I'd mention